src/components/molecules/Assignment/Assignment.tsx

```diff
@@ -167,9 +196,74 @@ const Assignment: FC<AssignmentProps> = ({
     }
   }, [t])
 
-  // TODO: shouldShowStartTimeFields no info about where to take this from yet
-  const shouldShowStartTimeFields =
-    project?.type_classifier_value?.project_type_config?.is_start_date_supported
+  const handleUpdateAssignment = useCallback(
```

Comment:
[BLOCKER]
Another thing (probably caused by the onSuccess callback of useAssignmentUpdate) is that it changes the order of assignments. After every update, the updated assignment goes to the bottom. So Teostaja 3 seems to have the assignment that should be the first in the order. Rather than adding the updated data to the end of array, update it in place.